openedx/frontend-app-learner-dashboard
# fix: force LMS url to reload when changed

Just change makes it possible if LMS_BASE_URL is updated by
 dynamic config to get the latest value.
 
 ## Testing: 

 While using tutor-mfe 
 1. add new this mfe to the list of MFEs [how](https://github.com/overhangio/tutor-mfe/tree/master#adding-new-mfes)
 2. Ensure [those settings](https://openedx.atlassian.net/wiki/spaces/OEPM/pages/3575906333/Learner+Home) are set, _can be done creating file plugin_
 3. If you use the `master` branch it will not work, while when using this `branch` shall fix it _from step 1 you can alter repository and versoin values` 
 
 ## More context 
 Fixes that are simlar to this 
  - gradebook openedx/frontend-app-gradebook/pull/290 
  - course authoring openedx/frontend-app-course-authoring/pull/389
